JavadocStyleGuide.md

```diff
@@ -0,0 +1,223 @@
+ * <p>
+ * This method always returns immediately, whether or not the
+ */
+```
+
+The inline tag `{@link URL}`, which converts to an HTML hyperlink pointing to the documentation for the URL class. 
+This inline tag can be used anywhere that a comment can be written, such as in the text following block tags:
+
+```java
+/**
+ * The url argument must specify an absolute <a href="#{@link}">{@link URL}</a>. The name
+ */
+```
+
+HTML tags for other block-level elements, such as `<ul>` or `<table>`, are not preceded with `<p>`.
+
+### 1.1.4 Symbols
+
+Entities for the less than symbol (<) and the greater than symbol (>) should be written as `&lt`; and `&gt;`. 
+Similarly, the ampersand (&) should be written as `&amp;`.
+
+```java
+/**
+ * The url argument must specify an absolute <a href="#{@link}">{@link URL}</a>. The name
+ */
+```
+### 1.1.5 Links
+
+The `{@link <class or method reference>}` tag is specifically used to link to the Javadoc of other classes and methods. 
+This is an inline tag that converts to an HTML hyperlink pointing to the documentation of the given class or method reference
+
+```java
+/**
+ * This &amp; that.
+ */
+```
+
+### 1.1.6 Code keywords, names, and variables
+
+Use `<code>...</code>` style for keywords and names including:
+- Java keywords
+- package names
+- class names
+- method names
+- interface names
+- field names
+- argument names
+- code examples (see section 1.3 below)
+
+```java
+/**
+ * Use the <code>Project</code> instance to configure the project.
+ */
+```
+
+### 1.1.7 Block tags
+
+Any of the standard "block tags" that are used appear in the order `@param`, `@return`, `@throw`s`, `@deprecated`, and these four types never appear with an empty description:
+
+```java
+/**
+ * @param  url  an absolute URL giving the base location of the image
+ * @param  name the location of the image, relative to the url argument
+ * @return      the image at the specified URL
+ * @see         Image
+ */
+```
+
+Full list of tags (in order):
+
+| # | Tag           | Usage                                                                          | Notes                                                                                      |
+|---|---------------|--------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|
+| 1 | `@param`      | Methods and constructors only                                                  | Include this if applicable                                                                 |
+| 2 | `@return`     | Methods only                                                                   | Include this if applicable                                                                 |
+| 3 | `@exception`  | Same as `@throws`                                                              | Include this if applicable                                                                 |
```

Comment:
Can we put `@throws` in the first column and `Same as @exception` in the second? I see `@throws` MUCH more often then `@exception` and I think that should be the "main" tag to use here

Reply:
Sounds good. Done.